src/ProjectTemplates/README.md

```diff
@@ -71,6 +71,14 @@ Then, use one of:
 previous step, it is NOT advised that you install templates created on your local machine using just
 `dotnet new -i [nupkgPath]`.
 
+#### Running Playwright tests
```

Comment:
Based on the title it seems like this is for running all playwright tests, but it's referring to only the `Blazor Template Tests`. Should the title be updated?